mentat/include_files.py

```diff
@@ -1,38 +1,17 @@
-import glob
-import logging
+import fnmatch
 import os
-from collections import defaultdict
+import re
 from pathlib import Path
-from typing import Any, Dict
+from typing import Any, Iterable, List, Set
 
-from mentat.code_feature import CodeFeature, parse_intervals
-from mentat.git_handler import get_non_gitignored_files
+from mentat.code_feature import CodeFeature
+from mentat.errors import PathValidationError
+from mentat.git_handler import get_git_root_for_path, get_non_gitignored_files
+from mentat.interval import parse_intervals
 from mentat.session_context import SESSION_CONTEXT
 
 
-def expand_paths(paths: list[Path]) -> tuple[list[Path], list[str]]:
```

Comment:
Same up here; why are we rewriting this?

Reply:
See https://github.com/AbanteAI/mentat/pull/286#issuecomment-1821996841